docs/pages/build-reference/variables.md

```diff
@@ -61,9 +63,92 @@ See the [eas.json reference](/build/eas-json.md) for more information.
 
 ## Using secrets in environment variables
 
-🚧 We are currently working on a secrets API that will allow developers to store generic encrypted secrets and selectively expose them to build jobs. This feature will be available before EAS Build graduates from preview.
+To provide your build jobs with access to values that are too sensitive to include in your source code and git repository, you can use "Secrets".
+
+These secrets are encrypted at rest and in transit, and are only decrypted in a secure environment by EAS servers.
+
+It is possible to create up to 100 account-wide secrets for each Expo account and 100 app-specific secrets for each app (you probably won't need that many). Account-wide, or "global" secrets will be exposed to every build environment across all of your apps. App-specific secrets apply to the app they're defined on, and override global secrets with the same name.
+
+You can manage secrets through the Expo website and EAS CLI.
+
+### Secrets on the Expo website
+
+To create secrets to use across all apps in an account, navigate to the "Secrets" tab under the [account settings](https://expo.io/settings/secrets):
```

Comment:
```suggestion
To create account-wide secrets, navigate to the "Secrets" tab under your account or organization's [settings](https://expo.io/settings/secrets):
```
(assuming the behavior is the same for personal accounts and orgs here)

Reply:
Yep, the behavior is the same, but the link would only go to the personal account